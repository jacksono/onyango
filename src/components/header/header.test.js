import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import Header from './header';
/* eslint no-undef:0 */
test('Header renders correctly', () => {
  const tree = renderer
    .create(
      <MuiThemeProvider>
        <Router>
          <Header />
        </Router>
      </MuiThemeProvider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
