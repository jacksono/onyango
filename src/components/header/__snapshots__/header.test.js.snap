// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="header"
>
  <h1>
    Notes Manager
  </h1>
  <div>
    <nav>
      <span>
        <a
          href="/auth/signIn"
          onClick={[Function]}
        >
          Sign In
        </a>
      </span>
      <span
        style={
          Object {
            "marginLeft": "10px",
          }
        }
      >
        <a
          href="/auth/register"
          onClick={[Function]}
        >
          Register
        </a>
      </span>
    </nav>
  </div>
</div>
`;
